analytics/filesystem/file_module.go

```diff
@@ -32,6 +33,15 @@ func (f *FileLogger) LogAuctionObject(ao *analytics.AuctionObject) {
 	f.Logger.Flush()
 }
 
+//Writes VideoObject to file
+func (f *FileLogger) LogVideoObject(vo *analytics.VideoObject) {
+	//Code to parse the object and log in a way required
+	var b bytes.Buffer
+	b.WriteString(jsonifyVideoObject(vo))
+	f.Logger.Debug(b.String())
+	f.Logger.Flush()
+}
+
```

Comment:
![image](https://user-images.githubusercontent.com/15863840/70836544-79abeb80-1dce-11ea-9a6f-6af80235ae7e.png)
Again, newly created function seems to not be covered by the test cases in contrast with other functions in the same file